starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,69 @@
+html{
+  margin: -20px 0px;
+}
+
+body{
+  text-align: center;
+font-family: Helvetica;
+border: 0px;
+padding: 0px 20px 30px;
+margin: 0;
+font-weight: lighter;
+}
+.H1{
+  background-image:url("/Users/ivanjorge/Prework/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
memorize these 2 lines: background size cover and background repeat no repeat.  You'll use this a million times